org.eclipse.gef.examples.logic/src/org/eclipse/gef/examples/logicdesigner/figures/LogicColorConstants.java

```diff
@@ -16,9 +16,10 @@
 
 public interface LogicColorConstants {
 
-	public final static Color andGate = new Color(null, 220, 70, 80);
-	public final static Color orGate = new Color(null, 0, 134, 255);
-	public final static Color xorGate = new Color(null, 240, 240, 40);
+	public final static Color andGate = new Color(null, 204, 61, 61);
```

Comment:
The first parameter is not needed anymore so as you are anyhow touching the code here you could remove all nulls in this interface.

Reply:
thx